docs/en/faq.md

```diff
@@ -1 +1,30 @@
 ## Frequently Asked Questions
+
+### TensorRT
+
+- "WARNING: Half2 support requested on hardware without native FP16 support, performance will be negatively affected."
+
+  Fp16 mode requires a device with full-rate fp16 support.
+
+- "error: parameter check failed at: engine.cpp::setBindingDimensions::1046, condition: profileMinDims.d[i] <= dimensions.d[i]"
+
+  When building an `ICudaEngine` from an `INetworkDefinition` that has dynamically resizable inputs, users need to specify at least one optimization profile. Which can be set in deploy config:
+
+  ```python
+  backend_config = dict(
+    common_config=dict(max_workspace_size=1 << 30),
+    model_inputs=[
+        dict(
+            input_shapes=dict(
+                input=dict(
+                    min_shape=[1, 3, 320, 320],
+                    opt_shape=[1, 3, 800, 1344],
+                    max_shape=[1, 3, 1344, 1344])))
+    ])
+  ```
+
+  The input tensor shape should be limited between `min_shape` and `max_shape`.
+
+- error: [TensorRT] INTERNAL ERROR: Assertion failed: cublasStatus == CUBLAS_STATUS_SUCCESS
```

Comment:
May put ""  to or remove ""  from all error logs

Reply:
[MMDetection](https://github.com/open-mmlab/mmdetection/blob/master/docs/en/faq.md) also use "" in FAQ.

Reply:
But the last item lack a `""`